docs/changelog/2_x.rst

```diff
@@ -0,0 +1,177 @@
+==
+v2
+==
+
+:edb-alt-title: EdgeDB v2—Something
+
+..
+   .. image:: images/v1_nova.jpg
+	   :width: 100%
+
+
+EdgeDB 2.0 was released on July XX, 2022. Read the announcement
+blog post `here <v2anno_>`_.
+
+We would like to thank our community for reporting issues and
+contributing fixes. You are awesome! ❤️
+
+
+
+===
+2.0
+===
+
+Major EdgeQL Features
+=====================
+
+* Add a ``GROUP`` statement for performming aggregation
+  (`RFC 1009 <group_>`_, `docs <_ref_eql_group>`). The
+  output of ``GROUP`` are free objects that contain the aggregated
+  elements as a computed link:
+
+  .. code-block:: edgeql-repl
+
+	  db> group Card { name } by .element;
+	  {
+		{
+		  key: {element: 'Earth'},
+		  grouping: {'element'},
```

Comment:
elements is the plural of element. So that's a bit confusing.
Does it pluralize the `element`? or is it just the api?

Reply:
I should change it to use an example where the key isn't named `element`

Reply:
changed it to the movies db